lib/peg.d.ts

```diff
@@ -607,6 +655,13 @@ export namespace compiler {
        * @param args Any arguments passed to the `Visitor`
        */
       one_or_more?(node: ast.Suffixed, ...args: any[]): any;
+      /**
+       * Default behavior: run visitor on `expression` and return it result
```

Comment:
It looks like the default implementation consumes the delimiter if it exists, then consumes the expression.